charts/jaeger/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.20.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
-version: 0.39.0
+version: 0.39.1
```

Comment:
Considering this will be a breaking change for anyone using Cassandra it would seem likely that the major version needs to be bumped.

Reply:
Today there will be no change in default storage setup, it is an advisory notice that storage should be explicitly configured to be resilient across the change of default storage setup. If this is too confusing I could remove that note altogether?

Reply:
I am happy with this change. It makes the README independent of any dependency chart changes.